modules/stanford_courses_importer/stanford_courses_importer.permissions.yml

```diff
@@ -0,0 +1,3 @@
+administer course importer settings:
```

Comment:
I'm guessing this was a left over of the H&S stuff too?

Reply:
removed.